### DISCUSSION TITLE
Integrating "Failures" developed by the PDF Association

### DISCUSSION THREAD

**Discussion by DuffJohnson** [w3c/wcag#3723] [Ideas (WCAG 2.x / WCAG 3)]
The PDF Association is developing a body of several hundred Techniques for PDF accessibility (see [example](https://pdfa.org/pdf-techniques-for-accessibility-a-new-model/)). A few of the forthcoming Failure examples are specific to PDF/UA-1, and are not WCAG failures. One example is PDF/UA-1's prohibition on skipped heading levels.

We could include a general statement on the front-end of our site acknowledging that not all PDF/UA failures are WCAG failures… or we could include some specific mention at the level of the individual technique…. or both. Or something else.

The PDF Accessibility LWG would like some input on how we should present such cases so we can work to minimize any speed-bumps on the way to the AGWG's hoped-for formal WCAG acceptance / inclusion of these new PDF Techniques alongside (or eventually, perhaps replacing) the existing PDF Techniques.

**Comment by mraccess77** [comment]
Both would be great - I like how current WCAG techniques that are not required indicate (Advisory) by the criterion that they are related to but only advisory for.  In addition, in understanding documents they are listed in a section marked Advisory.

Today there are several WCAG techniques for things such as consistent navigation, and other SC that don't apply the same as webpages etc. that are not clearly labeled and it creates confusion on whether they are best practice for WCAG or required.
